package.json

```diff
@@ -4,10 +4,12 @@
   "description": "Wordpress API Universal React Renderer",
   "main": "index.js",
   "scripts": {
-    "lint": "./node_modules/.bin/eslint src bin bootstrap test",
+    "lint": "eslint src bin bootstrap test --cache",
```

Comment:
Might be worth adding `.eslintcache` to `.gitignore` if it isn't covered by it already